src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -332,77 +364,18 @@ protected function getConstructor(array &$data, string $class, array &$context,
      *
      * @throws RuntimeException
      * @throws MissingConstructorArgumentsException
+     *
+     * @deprecated
      */
     protected function instantiateObject(array &$data, string $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes, string $format = null)
```

Comment:
Replaced by `Instantiator::instanciate` method. Should I add a message on the deprecation notice ?

Reply:
Fixed